src/custodia/store/encgen.py

```diff
@@ -60,13 +68,29 @@ def get(self, key):
         try:
             jwe = JWE()
             jwe.deserialize(value, self.mkey)
-            return jwe.payload.decode('utf-8')
         except Exception as err:
             self.logger.error("Error parsing key %s: [%r]" % (key, repr(err)))
             raise CSStoreError('Error occurred while trying to parse key')
+        value = jwe.payload.decode('utf-8')
```

Comment:
we shouldn't keep this check inside the try/except block? Since this decode method is part of the parse process?